solr/modules/hadoop-auth/src/test/org/apache/solr/security/hadoop/KerberosTestServices.java

```diff
@@ -198,10 +198,18 @@ public AppConfigurationEntry[] getAppConfigurationEntry(String name) {
    * These Locales don't generate dates that are compatible with Hadoop MiniKdc. See LocaleTest.java
    * and https://issues.apache.org/jira/browse/DIRKRB-753
    */
-  static final List<String> incompatibleLanguagesWithMiniKdc =
-      Arrays.asList(
-          "mzn", "ps", "mr", "uz", "ks", "bn", "my", "sd", "pa", "ar", "th", "dz", "ja", "ne",
-          "ckb", "fa", "lrc", "ur", "ig", "sat", "mni", "sa", "as");
+  static final List<String> incompatibleLanguagesWithMiniKdc = getIncompatibleLanguagesWithMiniKdc();
+
+  private static List<String> getIncompatibleLanguagesWithMiniKdc() {
+    List<String> locales = Arrays.asList(
+        "mzn", "ps", "mr", "uz", "ks", "bn", "my", "sd", "pa", "ar", "th", "dz", "ja", "ne",
+        "ckb", "fa", "lrc", "ur", "ig", "sat", "mni", "sa", "as");
+    if (Runtime.version().feature() >= 20) {
```

Comment:
If these locales don't exist in `<20`, then do we need this check, I imagine it would be safe to add them to the list in all cases

Reply:
Yea I misread the logic in LocaleTest and thought needed to separate this.

Reply:
addressed in 51c6595